js/app.js

```diff
@@ -13,13 +13,17 @@ jQuery( document ).ready(function() {
     }
   }
 
-  var regionNameArea = jQuery('#region_name');
+  var regionNameArea = jQuery('.region_name_container');
+
+  var parentRegionArea = jQuery('.parent_region_container');
 
   var stpcodeArea = jQuery('#st_pcode');
 
   var tspcodeArea = jQuery('#ts_pcode');
 
-  var totalPopulation = jQuery('#total_population')
+  var totalPopulation = jQuery('#total_population');
+
+  var currentRegionProperties = 'Myanmar';
```

Comment:
@DragonVirus is this necessary? Does **js/app.js** only apply to the template for the burmese dashboards? If this file contains general JS logic, such country-specific values should not be hardcoded.